libportal/notification.c

```diff
@@ -94,7 +167,8 @@ call_done (GObject *source,
  *
  * - title `s`: a user-visible string to display as title
  * - body `s`: a user-visible string to display as body
- * - icon `v`: a serialized icon (in the format produced by [method@Gio.Icon.serialize])
+ * - icon `v`: a serialized icon (in the format produced by [method@Gio.Icon.serialize]
```

Comment:
This needs to be a defined standard rather than explicitly requiring a library.

Reply:
The serialized icon is a requirement for v1 of the portal which is unfortunate. I guess we could take an arbitrary piece of memory for this API though and convert it to the serialized icon when only v1 is supported.

Reply:
@davidedmundson Libportal provides GIO-style API for the portals so it's fine to depend on GIO.

@swick not sure what you mean? do you suggest to change the API?

Reply:
I think the point I was trying to make is that we can now send an arbitrary blob of data for icons and sounds without having to serialize them. This API doesn't allow for that.

Let's not get hung up on this and try to get the existing changes here into shape to get it merged.

Reply:
ack, I didn't read the repo name properly. This is a non-issue.